Jenkinsfile

```diff
@@ -39,6 +39,16 @@ properties(
           description: 'Contains the CI_MESSAGE for a message bus triggered build.',
           name: 'CI_MESSAGE'
         )
+	string(
+	  defaultValue: '',
+	  description: 'Hostname of the preprovisoined host.',
+	  name: 'PREPROVISIONED_HOST'
+	)
```

Comment:
You need a comma after the `)` since this is a list of parameters that is part of a bigger `[]` block